medusa/providers/torrent/html/horriblesubs.py

```diff
@@ -102,7 +102,7 @@ def parse(self, data, mode):
                         if mode != 'RSS':
                             # keep the date and strip the rest
                             pubdate_raw = row.find('td', class_='rls-label').get_text()[1:9]
-                            pubdate = self.parse_pubdate(pubdate_raw)
+                            pubdate = self.parse_pubdate(pubdate_raw, timezone='America/Los_Angeles')
```

Comment:
https://tools.ietf.org/html/rfc7231#section-7.1.1.2

It's not the server time.